app/src/main/java/org/kiwix/kiwixmobile/search/AutoCompleteAdapter.java

```diff
@@ -34,17 +38,16 @@
 import org.kiwix.kiwixmobile.data.ZimContentProvider;
 import org.kiwix.kiwixmobile.utils.SharedPreferenceUtil;
 
-public class AutoCompleteAdapter extends ArrayAdapter<String> implements Filterable {
-
+public class AutoCompleteAdapter extends RecyclerView.Adapter<AutoCompleteAdapter.ViewHolder> implements Filterable {
   @Inject JNIKiwix currentJNIReader;
   @Inject
   SharedPreferenceUtil sharedPreferenceUtil;
-  private List<String> mData;
+  List<String> mData;
   private KiwixFilter mFilter;
   private Context context;
 
   public AutoCompleteAdapter(Context context) {
-    super(context, android.R.layout.simple_list_item_1);
+    //super(context, android.R.layout.simple_list_item_1);
```

Comment:
No need to comment just delete this line, we want to make our codebase super clean.

Reply:
Yes. Thanks.